contrib/windows/build-installer.iss

```diff
@@ -79,17 +70,18 @@ AppName={#AppName}
 AppVersion={#AppVersion}
 AppPublisher=Julia Language
 AppPublisherURL=https://julialang.org
+AppSupportURL=https://julialang.org
+AppUpdatesURL=https://julialang.org
 AppCopyright=Copyright 2009-{#CurrentYear}; Julia Langage
 VersionInfoDescription=Julia Installer
-UsePreviousPrivileges=no
-PrivilegesRequired=lowest
 PrivilegesRequiredOverridesAllowed=commandline
 WizardStyle=modern
 Compression=lzma2/ultra
 SolidCompression=yes
-SetupIconFile={#AppHomeFiles}\contrib\windows\julia.ico
-DefaultDirName={autopf}\{#AppName}\{#AppName}-{#AppVersion}
-DefaultGroupName="{#AppName} {#AppVersion}"
+DefaultDirName={autopf}\{#AppNameLong}
+UsePreviousPrivileges=no
+PrivilegesRequired=lowest
+DefaultGroupName="{#AppNameLong}"
```

Comment:
Instead of `DefaultGroupName="{#AppNameLong}"` , could this be just `DefaultGroupName="{#AppName}" which would fix https://github.com/JuliaLang/julia/issues/34867 ?

Reply:
Note please see the linked issues. We shouldn't install the Julia Icon into a folder structure as a general recommendation by Windows.

The changes in this PR install the Julia x.y ver, directly into the menu.